CHANGELOG.yml

```
@@ -30,6 +30,15 @@ docDescription: >-
   environments, access to instantaneous feedback loops, and highly
   customizable development environments.
 items:
+  - version: 2.20.0
+    date: (TBD)
+    notes:
+      - type: feature
+        title: Add support for Argo Rollout workloads.
+        body: >-
+          Telepresence now supports Argo Rollout workloads.
+          It is recommended to set the following annotation `telepresence.getambassador.io/inject-traffic-agent: disabled`
```

```suggestion
          It is recommended to set the following annotation <code>telepresence.getambassador.io/inject-traffic-agent: enabled</code>
```
It should be enabled, and believe it or not, but this text is html, not markdown.